ProcessList.c

```diff
@@ -646,7 +660,7 @@ void ProcessList_scan(ProcessList* this, bool pauseProcessUpdate) {
       if (p->tombStampMs > 0) {
          // remove tombed process
          if (this->monotonicMs >= p->tombStampMs) {
-            ProcessList_remove(this, p);
+            ProcessList_removeIndex(this, p, i);
```

Comment:
This is still quadratic for large numbers of processes …

Reply:
Yep, I forgot about the vector reclaiming space via `memmove` as it was not prominent in profiles `sudo perf record -F 99 -g`. It was mostly `Vector_indexOf` that caught my attention.